packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -53,7 +53,17 @@ export const runC3 = async ({
 }: RunnerConfig) => {
 	const cmd = "node";
 	const args = ["./dist/cli.js", ...argv];
-	const proc = spawn(cmd, args);
+	const proc = spawn(cmd, args, {
+		env: {
+			...process.env,
+			// The following env vars are set to ensure that package managers
+			// do not use the same global cache and accidentally hit race conditions.
```

Comment:
Is this something we want to do in real world usage or just in test? I don't see any harm in it just curious if there's a real world use case outside of test that will benefit from it

Reply:
I did consider this but I think we should not be taking over general users cache locations.
If they want, they can set these environment variables to get the same effect. Giving them control.